### DISCUSSION TITLE
Strange terminal behavior with docker alpine shell

### DISCUSSION THREAD

**Discussion by berkant** [community/community#3466] [Codespaces] [answered]
Hi.

I'm having a strange terminal behavior when `exec sh`'ing on docker alpine containers. Entering commands and responses simply won't update the terminal window. Only resizing the terminal makes them appear.

You can reproduce with:
```terminal
docker run --rm -it alpine sh
```
and running some commands.

**Comment by bartr** [comment]
I ran into the same issue.

The results are actually there, you just can't see them (highlight in the terminal and copy)

running apk update displays correctly

running docker run -i --rm alpine (no -t) works correctly, but you don't get tty

docker run -it --rm alpine ls -alF works correctly

running with -d and using docker exec -it works correctly

**Comment by Chuxel** [comment, marked as answer]
This may be https://github.com/microsoft/vscode/issues/110232

**Reply by bartr** [reply]
I verified the same behavior using VS Code over SSH to an Ubuntu VM

I just create an image from alpine, install bash and set CMD to /bin/bash -l  It's a bit of a hack, but, works for me
